app/views/surveys/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="d-flex align-items-center justify-content-between mb-5">
+  <h1 class="h3 mb-0 pb-0 font-weight-bold">Umfragen</h1>
+  <%= link_to new_survey_path, class: "btn btn-sm btn-primary shadow-sm ml-3" do %>
+    <i class="fas fa-plus fa-sm text-white-50"></i>
+    Neue Umfrage erstellen
+  <% end %>
+</div>
+
+<div class="table-responsive pb-4">
+  <table class="table data_table bg-white">
+    <thead>
+      <tr>
+        <th scope="col">ID</th>
+        <th scope="col">Titel</th>
+        <th scope="col">Änderungsdatum</th>
+        <th scope="col">Erstellungsdatum</th>
```

Comment:
we could think of more suitable infos in the table. the dates can make sense a well as the number of votes for example? there are no requirements or acceptance criteria about this. it is on us.

Reply:
lets keep it out for now as it is not part of the ticket, and does not have structural implications for other features